ReactApp/car_rental_react/src/routes/CarReserve.js

```diff
@@ -1,154 +1,121 @@
 import { Button } from "reactstrap";
 import axios from "axios";
+import { useLocation } from "react-router-dom";
+import { useAuth } from "../provider/authContext";
 
 const CarReserve = () => {
-  function signIn() {
-    alert("Feature not yet implemented!");
-  }
+  //const { isSignedIn, user, logOut } = useAuth();
+  //console.log(isSignedIn, user);
 
-  function submitRental() {
-    // first create a customer from the details inputted
-    const customer_fields = document.getElementsByClassName("rental-input");
-    var customer_input = [];
-    for (let index = 0; index < customer_fields.length; index++) {
-      customer_input.push(customer_fields[index].value);
-    }
+  // get info from car selection
+  const location = useLocation();
+  console.log(location.state);
+
+  //get car information
+  var car_model;
+  var car_type;
+  var car_id;
+  axios
+    .get("/api/cars/" + location.state.carid + "/")
+    .then((res) => {
+      // get model of car
+      car_model = res.data.manufacturer + " " + res.data.model;
+      let element = document.getElementById("cartype");
+      element.innerHTML = "Car Type: " + car_model;
+
+      // get car type and car id
+      car_type = res.data.car_type;
+      car_id = res.data.car_id;
+
+      // update estimated price
+      let price_element = document.getElementById("price");
+      price_element.innerHTML =
+        "Estimated Price: " + location.state.estimatedcost + "$";
+    })

```

Comment:
Instead of changing innerHTML, which is considered bad practice in React, can you use state to modify the text?